runbotics/runbotics-desktop/src/core/bpm/CamundaExtension.js

```diff
@@ -50,8 +50,10 @@ export function Camunda(activity, processEnvironment) {
         //     // behaviour.Service = ServiceImplementation
         // }
         if (activity.type === 'bpmn:ServiceTask') {
+            const disabled = activity.behaviour.disabled;
             activity.environment.runbotic = {
-                disabled: activity.behaviour.disabled,
+                disabled,
+                ...(!disabled && { processOutput: activity.behaviour.processOutput })
```

Comment:
is it necessary?

Reply:
I think this is the easiest way to pass form property from each activity of process